runner/src/server/plugins/engine/pageControllers/RepeatingFieldPageController.ts

```diff
@@ -169,7 +175,9 @@ export class RepeatingFieldPageController extends PageController {
       }
 
       if (typeof partialState !== "undefined") {
-        return this.summary.getRouteHandler(request, h);
+        return h.redirect(
+          `?view=${view ?? "summary&"}${form_session_identifier}`
```

Comment:
Does the view have a & attached to the end of it? if not we might need to consider that, just a q

Reply:
its a query string so the and represents the more parts